src/cli/pm_trusted_command.zig

```diff
@@ -169,16 +169,17 @@ pub const TrustCommand = struct {
         Global.crash();
     }
 
-    fn printErrorZeroUntrustedDependenciesFound(trust_all: bool, packages_to_trust: []const string) void {
+    fn printErrorZeroUntrustedDependenciesFound(trust_all: bool, packages_to_trust: []const string) noreturn {
         Output.print("\n", .{});
         if (trust_all) {
-            Output.errGeneric("0 scripts ran. This means all dependencies are already trusted or none have scripts.", .{});
+            Output.warn("0 scripts ran. This means all dependencies are already trusted or none have scripts.", .{});
         } else {
-            Output.errGeneric("0 scripts ran. The following packages are already trusted, don't have scripts to run, or don't exist:\n\n", .{});
+            Output.warn("0 scripts ran. The following packages are already trusted, don't have scripts to run, or don't exist:\n\n", .{});
             for (packages_to_trust) |arg| {
                 Output.prettyError(" <d>-<r> {s}\n", .{arg});
             }
         }
+        Global.exit(0);
```

Comment:
Whats the rationale? This is a breaking change

Reply:
its exactly the same behavior as before, i just moved this call from the callers into this function

Reply:
oh i see; it not being an error. it didnt seem like something that should be an error condition and i consulted with dylan

Reply:
the runner script runs this command and having this be an error makes it indistinguishable from having scripts that failed/crashed vs there were none in the first place

Reply:
https://github.com/search?q=%22bun+pm+trust+--all%22&type=code looks like there's not a ton of in-the-wild usage and there's even multiple counts of users doing `|| true` potentially to obtain this behavior on their own